VigneshApp/src/app/product-list-page/product-list-page.component.html

```diff
@@ -0,0 +1,22 @@
+<div
+    class="min-w-6/12 min-h-3/6 border border-solid border-lime-600 rounded-3xl shadow-lime-400 shadow-sm flex flex-col justify-center px-4 py-10">
+    <div class="flex m-4 items-center">
+        <p class="font-bold px-4">Add Product:</p>
```

Comment:
Best practices use label here instead of p

Reply:
sure will update now